src/background/utils/network.ts

```diff
@@ -42,14 +42,24 @@ class LimitedCacheStorage {
 
     private bytesInUse = 0;
     private records = new Map<string, CacheRecord>();
+    private hasAlarm = false;
```

Comment:
```suggestion
    private alarmIsActive = false;
```

Seems like a more descriptive variable to give.

Reply:
Done!